docs/SetupDatabaseForAWS.md

```diff
@@ -0,0 +1,20 @@
+# Set up a database for Scalar DB/Scalar DL deployment on AWS
+
+This guide explains how to set up a database for Scalar DB/Scalar DL deployment on AWS.
+
+## DynamoDB
+
+By default, Amazon DynamoDB is available for all AWS users. You do not need to set up anything manually.
+
+### Steps
+
+* For configuring DynamoDB properties like autoscale, Read and WriteRequest Unit,
+  * For Scalar DL, [Scalar DL schema loader](https://github.com/scalar-labs/scalardl-schema-loader) or the Scalar DL schema loader [helm charts](https://github.com/scalar-labs/helm-charts/tree/main/charts/schema-loading) will help you to configure properties like autoscale, Read and WriteRequest Unit for DynamoDB tables.
+  * For Scalar DB, [Scalar DB schema loader](https://github.com/scalar-labs/scalardb/tree/master/schema-loader/) will help you to configure properties like autoscale, Read and WriteRequest Unit for DynamoDB tables.
+* (Optional) You can scale the throughput of DynamoDB by setting the `dynamoBaseResourceUnit` value (which applies to all the tables) in the `schema-loading-custom-values.yaml` file.
```

Comment:
The option is available for both scalar DB/DL schema loaders (there is a --ru option for dynamodb and cosmos db).
But it may cause a bit of confusion since it is specific for scalar DL kubernetes deployment. So I have removed it.